### ISSUE TITLE
 Cannot Reproduce the result

### ISSUE THREAD

**Issue by swzhang5** [hasanirtiza/Pedestron#120]
Hi, I am trying to reproduce the results of 'CityPersons->CityPersons' of Cascade R-CNN in the paper, which is 11.2 for Reasonable MR. 
I use 7 2080Ti, because the GPU memory cannot support 2 imgs per GPU, I set it to 1 img/gpu, and the lr to 0.01. I trained the network for 20 epochs, and the best result I got is 12.23. Can you help me reproduce the result? Thanks a lot.

**Comment by hasanirtiza** [comment]
We have managed to get close to 11.2 with 2 GPUS. I would may be start with the default learning rate and then play around it. May be this might help #115.

**Comment by swzhang5** [comment]
> We have managed to get close to 11.2 with 2 GPUS. I would may be start with the default learning rate and then play around it. May be this might help #115.

Could you share the setting for 2 GPUS please? Thank you.

**Comment by hasanirtiza** [comment]
> > We have managed to get close to 11.2 with 2 GPUS. I would may be start with the default learning rate and then play around it. May be this might help #115.
> 
> Could you share the setting for 2 GPUS please? Thank you.

Did you try the default settings ? As far as I remember we used the default settings.
